src/ray/core_worker/transport/actor_task_submitter.cc

```diff
@@ -732,10 +738,11 @@ void ActorTaskSubmitter::HandlePushTaskReply(const Status &status,
 
         GetTaskFinisherWithoutMu().CompletePendingTask(
             task_id, reply, addr, reply.is_application_error());
-      }
-      // last failure = Actor death, but we still see the actor "alive" so we optionally
-      // wait for a grace period for the death info.
-      else if (RayConfig::instance().timeout_ms_task_wait_for_death_info() != 0) {
+
+      } else if (RayConfig::instance().timeout_ms_task_wait_for_death_info() != 0) {
```

Comment:
curious what's the issue here?

Reply:
I fixed this issue based on the lint error:
```
src/ray/core_worker/transport/actor_task_submitter.cc:738: An else should appear on the same line as the preceding } [whitespace/newline] [4].
```

Reply:
Cool thanks for the explanation!